tests/pdarray_creation_test.py

```diff
@@ -419,6 +419,10 @@ def test_full(self):
         string_len_full = ak.full("5", 5)
         self.assertEqual(5, len(string_len_full))
 
+        strings_full = ak.full(5, "test")
+        self.assertIsInstance(strings_full, ak.Strings)
+        self.assertEqual(5, len(strings_full))
```

Comment:
It may also be beneficial here to verify that all of the values in the resulting Strings object are the value provided.

Reply:
Looks like this test didn't get added